terms/generativemodels_pm.tex

```diff
@@ -0,0 +1,80 @@
+\chapter{Generative Models and Networks}
+\label{ch:Generative Models and Networks}\index{Generative Models and Networks}
+
+\subsection{Definition}
+A generative model is an unsupervised statistical model that can be used to generate similar samples from the learning of observable examples \citet{shin:2017}.
+These types of models make use of Naive Bayes rules in joint probability to calculate Equation \ref{p(y,x)}:
+
+\begin{equation}\label{p(y,x)}
+p(y,x) = p(x)p(y|x)
+\end{equation}
+
+With \textit{x} being the input feature and \textit{y} being the given class \citet{ng:2002}.
+
+Generative models are a concept which are currently limited in use due to the challenge of productively training them, 
+where article such as \citet{goodfellow:2014} state the difficulty of 
+finding the best way to train them as the probabilistic computation required tends to be hard to gather the linear units in a generative context.
+
+
+
+They are still sought after due to their heavy use in computer vision such as image caption generation \citet{lin:2014,touretzky:1996}; projects related to speech and language for example speech synthesis \citet{ou:2012} and conversation with dialogue generation \citet{sordoni:2015}. Generational models have the capability of data generation from the given sample with the understanding of the structure of the input data even without labels \citet{tu:2007}.
+
+
+\begin{figure}
+  \centering
+  \includegraphics[width=1\linewidth]{graphics/generational_models/gm.png}
+
+  \caption[Caption for GAN]{The layout of a generative model that shows the production of a generated product that is a variation of the original image space \protect\footnotemark. }
+  \label{fig:GN}
+\end{figure}
+\footnotetext{www.blog.openai.com/generative-models/}
+\section{Generative Adversarial Networks}
+
+Generative Adversarial Networks \index{Generative Adversarial Networks} (GAN) are a type of generative approach that makes use of the creation of a network which consists of, the generator model which maps the latent space to the input space \textit{x} and with the introduction of a discriminative model. It will be used to classify the generated data sample of the first model and calculates the probability of the sample that it came from the product of the first model or the actual training data. These models  are trained simultaneously where the performance of the generative model is based on the probability of the discriminative model to make a mistake  \citet{goodfellow:2014}. 
+
+GAN is defined by the following objective function:
```

Comment:
There should be no text stuff (textasciitilde) in equations !!!! typseset only maths.